Jenkinsfile

```diff
@@ -6,6 +7,28 @@ pipeline {
     stages {
         stage('Build') {
             steps {
+                populateGlobalVariables()
+                notifySlack("", 'fuse-java-builds', [
+                    [
```

Comment:
Is there any way of reducing the duplication in these calls? At a minimum it would be nice to create a _getFields_ method that returns the fields array.